content/docs/user-guide/experiment-management/index.md

```diff
@@ -52,6 +58,8 @@ roll back to the baseline. The basic workflow goes like this:
 - Make the selected experiment persistent by committing its results to Git. This
   cleans the slate so you can repeat the process.
 
+> Note that these features won't work if you don't use Git to version your code.
```

Comment:
move it before the list? (feels a bit too late at this stage)

Reply:
OK sure, moved.